doc/dev/add_missing_obj.rst

```diff
@@ -8,13 +8,14 @@ Object position on the slit from slitmask design
 Version History
 ---------------
 
-=========   ================   =========== ===========
-*Version*   *Author*           *Date*      ``PypeIt``
-=========   ================   =========== ===========
-1.0         Debora Pelliccia   02 Apr 2021  1.3.4dev
-1.1         Debora Pelliccia   28 Jul 2021  1.4.3dev
-1.3         Debora Pelliccia   21 Oct 2021  1.6.1dev
-=========   ================   =========== ===========
+=========   =================== =========== ===========
+*Version*   *Author*            *Date*      ``PypeIt``
+=========   =================== =========== ===========
+1.0         Debora Pelliccia    02 Apr 2021  1.3.4dev
+1.1         Debora Pelliccia    28 Jul 2021  1.4.3dev
+1.3         Debora Pelliccia    21 Oct 2021  1.6.1dev
+1.4         J. Xavier Prochaska 21 Oct 2021  1.6.1dev
```

Comment:
Maybe you want to change Date and PypeIt version here

Reply:
Indeed! :)